selfdrive/ui/ui.cc

```diff
@@ -96,7 +96,7 @@ void update_model(UIState *s,
   int max_idx = get_path_length_idx(lane_lines[0], max_distance);
   for (int i = 0; i < std::size(scene.lane_line_vertices); i++) {
     scene.lane_line_probs[i] = lane_line_probs[i];
-    update_line_data(s, lane_lines[i], 0.025 * scene.lane_line_probs[i], 0, &scene.lane_line_vertices[i], max_idx);
+    update_line_data(s, lane_lines[i], 0.025, 0, &scene.lane_line_vertices[i], max_idx);
```

Comment:
```suggestion
    update_line_data(s, lane_lines[i], 0.025 * fmin(scene.lane_line_probs[i], 0.5), 0, &scene.lane_line_vertices[i], max_idx);
```

it looks pretty ugly without something, maybe clipping to half is fine.

before:

![image](https://github.com/commaai/openpilot/assets/25857203/21b54362-259f-4170-8b6b-bf1a2de884b6)

after:

![image](https://github.com/commaai/openpilot/assets/25857203/5de832b8-6c43-4b9f-9dea-f07366592bcd)

Reply:
This likely only worked because it likely reduced triangles near the end of the ll